application/templates/static/contributors.html

```diff
@@ -0,0 +1,152 @@
+{% load static %}
+{% include "../snippets/head.html" %}
+{% include "../snippets/header.html" %}
+
+<div id="goals-page" class="container">
+	{% include "../snippets/page-menu.html" %} 
+	<section class="page-content">
+		<h1 class="section-header dark-blue credit-divider">Contributors</h1>
+		<div class="columns">
+			<ul class="source-sans center-list-25">
+				<li>Anna Waldman-Brown <small><i>&lt;annawab  at fabfolk.com&gt;</i></small></li>
+				<li>Albie Brown <small><i>&lt;albiejbrown at  gmail.com&gt;</small></i></li>
+				<li>Rusty Grove <small><i>&lt;rusty  at  blatypus.com&gt;</small></i></li>
+				<li>Chinmayi SK <small><i>&lt;chinmayisk at  gmail.com&gt;</small></i></li>
+				<li>Joséphine Lesaffre <small><i>&lt;josephine.lesaffre  at  gmail.com&gt;</small></i></li>
+				<li>Will Newman <small><i>&lt;wnewman88  at  gmail.com&gt;</small></i></li>
+				<li>Lyla Fischer <small><i>&lt;lylafisch  at  gmail.com&gt;</small></i></li>
+				<li>Jodi Huang <small><i>&lt;jodihuang1  at  gmail.com&gt;</small></i></li>
+				<li>Becky Button <small><i>&lt;einsteinunicorn  at  gmail.com&gt;</small></i></li>
+				<li>Maylan VN <small><i>&lt;maylanvn  at  gmail.com&gt;</small></i></li>
+				<li>David García </li>
+			</ul>
+		</div>
```

Comment:
Let's put the code contributors in a "Contibutors.txt" file in the code repo, rather than on the contributors page on the site (because that tends to be more corporate rather than person-to-person)